js/digifinex.js

```diff
@@ -662,14 +662,19 @@ module.exports = class digifinex extends Exchange {
         //         "fee": 0.096,
         //         "fee_currency": "USDT",
         //         "timestamp": 1499865549,
-        //         "side": "buy",
+        //         "side": "buy", // or "side": "sell_market"
         //         "is_maker": true
         //     }
         //
         const id = this.safeString (trade, 'id');
         const orderId = this.safeString (trade, 'order_id');
         const timestamp = this.safeTimestamp2 (trade, 'date', 'timestamp');
-        const side = this.safeString2 (trade, 'type', 'side');
+        let side = this.safeString2 (trade, 'type', 'side');
+        let type = undefined;
+        if (side.indexOf ('_market') >= 0) {
+            side = side.replace ('_market', '');
+            type = 'market';
```

Comment:
@kroitor since when do we have a type 'market'? 
I never saw this type before, only 'trade'